packages/notices/src/store/test/actions.js

```diff
@@ -0,0 +1,188 @@
+		} );
+
+		it( 'should yields actions when options passed', () => {
+			const id = 'my-id';
+			const context = 'foo';
+			const options = {
+				id,
+				isDismissible: false,
+				context: 'foo',
+			};
+
+			const result = createNotice( status, content, options );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'SPEAK',
+				message: content,
+			} );
+
+			expect( result.next().value ).toEqual( {
+				type: 'CREATE_NOTICE',
+				context,
+				notice: {
+					id,
+					status,
+					content,
+					isDismissible: false,
+					actions: [],
+				},
+			} );
+		} );
+	} );
+
+	describe( 'createSuccessNotice', () => {
+		it( 'should return action', () => {
+			const content = 'my message';
+
+			const result = createSuccessNotice( content );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'SPEAK',
+				message: content,
+			} );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'CREATE_NOTICE',
+				context: DEFAULT_CONTEXT,
+				notice: {
+					status: 'success',
+					content,
+					isDismissible: true,
+					id: expect.any( String ),
+				},
+			} );
+		} );
+	} );
+
+	describe( 'createInfoNotice', () => {
+		it( 'should return action', () => {
+			const content = 'my message';
+
+			const result = createInfoNotice( content );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'SPEAK',
+				message: content,
+			} );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'CREATE_NOTICE',
+				context: DEFAULT_CONTEXT,
+				notice: {
+					status: 'info',
+					content,
+					isDismissible: true,
+					id: expect.any( String ),
+				},
+			} );
+		} );
+	} );
+
+	describe( 'createErrorNotice', () => {
+		it( 'should return action', () => {
+			const content = 'my message';
+
+			const result = createErrorNotice( content );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'SPEAK',
+				message: content,
+			} );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'CREATE_NOTICE',
+				context: DEFAULT_CONTEXT,
+				notice: {
+					status: 'error',
+					content,
+					isDismissible: true,
+					id: expect.any( String ),
+				},
+			} );
+		} );
+	} );
+
+	describe( 'createWarningNotice', () => {
+		it( 'should return action', () => {
+			const content = 'my message';
+
+			const result = createWarningNotice( content );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'SPEAK',
+				message: content,
+			} );
+
+			expect( result.next().value ).toMatchObject( {
+				type: 'CREATE_NOTICE',
+				context: DEFAULT_CONTEXT,
+				notice: {
+					status: 'warning',
+					content,
+					isDismissible: true,
+					id: expect.any( String ),
+				},
+			} );
+		} );
+	} );
+
+	describe( 'removeNotice', () => {
+		it( 'should return action', () => {
+			const id = 'id';
+
+			expect( removeNotice( id ) ).toEqual( {
+				type: 'REMOVE_NOTICE',
+				id,
+				context: DEFAULT_CONTEXT,
+			} );
+		} );
+
+		it( 'should return action with context', () => {
```

Comment:
Super nit: we can make this statement more explicit - `...action with *custom* context`

Reply:
> Super nit: we can make this statement more explicit - `...action with *custom* context`

Updated in rebased 009b8e916b3ae9fac32af65261713feab454097f.